packages/medusa/src/api/routes/admin/reservations/get-reservation.ts

```diff
@@ -55,9 +55,12 @@ export default async (req, res) => {
   const inventoryService: IInventoryService =
     req.scope.resolve("inventoryService")
 
-  const [reservations, count] = await inventoryService.listReservationItems({
-    id,
-  })
+  const [reservations, count] = await inventoryService.listReservationItems(
```

Comment:
todo: missing manager, it will create a transaction under the hood and same issue. With the manager we ensure to use the regular manager without transaction under the hood